bitnami/mariadb-galera/values.yaml

```diff
@@ -142,15 +157,22 @@ service:
     publishNotReadyAddresses: true
 ## Pods Service Account
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+## @param serviceAccount.create Specify whether a ServiceAccount should be created
+## @param serviceAccount.name Name of the service account to use. If not set and create is true, a name is generated using the fullname template.
+## @param serviceAccount.automountServiceAccountToken Automount service account token for the server service account
+## @param serviceAccount.annotations Annotations for service account. Evaluated as a template. Only used if `create` is `true`.
 ##
 serviceAccount:
-  ## @param serviceAccount.create Specify whether a ServiceAccount should be created
-  ##
   create: false
-  ## @param serviceAccount.name The name of the ServiceAccount to create
-  ## If not set and create is true, a name is generated using the common.names.fullname template
-  ##
   name: ""
+  automountServiceAccountToken: true
```

Comment:
Does MariaDB Galera makes uses of the K8s API by default? If not, this should be set to "false" by default.